auth/database/migrations/2021_07_28_130831_create_user_roles.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateUserRoles extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('user_roles', function (Blueprint $table) {
+            $table->id();
+            $table->string('role');
```

Comment:
Oh, uh ... that's a good question. Well, firstly I'm not sure if we are even keeping this implementation (we might move to storing roles in the api db). So ... perhaps Ill wait on that. In my head this was only for checking role in code (if role == whatever then do whatever) rather than to display on the screen. 

As per usual @tristan-orourke to confirm :)

Reply:
Hmm. I would have also just gone with string, but you could make the argument that it should have the whole lookup table treatment: immutable key, translatable name, translatable description. 
Buuuuut I think we'll end up moving roles to the graphql api anyway, so it doesn't really matter now. 🤷